lib/Models/TimeSeriesStack.js

```diff
@@ -66,8 +67,9 @@ TimeSeriesStack.prototype.addLayerToTop = function(item) {
  */
 TimeSeriesStack.prototype.removeLayer = function(item) {
     var index = this._layerStack.indexOf(item);
-
-    this._layerStack.splice(index, 1);
+    var layerStack = this._layerStack.slice(); // Make a copy to trigger the knockout subscription on _layerStack.
```

Comment:
Good point. Actually I think it was calling `updateForNewTopLayer(this);` from the constructor - not just the `topLayer` subscription - that was the key part.  I'll check it out and remove this guff if it's not needed.